modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryExtensionProvider.java

```diff
@@ -0,0 +1,342 @@
+/**
```

Comment:
I think you added this files accidentally

Reply:
Yes, I did... thanks god you realised (and the source formatter xD)